.github/workflows/ci-run-e2e.yml

```diff
@@ -0,0 +1,59 @@
+name: CI Run e2e
+
+on:
+  repository_dispatch:
+    types: [run-e2e-command]
+
+env:
+    THREE_SCALE_PREVIEW_USER_KEY: ${{ secrets.THREE_SCALE_PREVIEW_USER_KEY }}
+    RECOMMENDER_REFRESH_TOKEN: ${{ secrets.RECOMMENDER_REFRESH_TOKEN }} 
+    REGISTERED_USER_UUID: ${{secrets.REGISTERED_USER_UUID }} 
+    SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
+
+jobs:
+    Run-e2e-build:
+      runs-on: ubuntu-latest
+      steps:
+        - uses: actions/checkout@v2
+          if: github.event_name == "repository_dispatch"  && github.event.action == 'run-e2e-command'
+        - run: |
+            git fetch origin +refs/pull/${{ github.event.client_payload.github.payload.issue.number }}/merge
+            git checkout FETCH_HEAD
+        - name: Prepare Environment
+          run: |
+            F8A_API_URL=${F8A_API_URL:-https://recommender.api.prod-preview.openshift.io};
+            F8A_JOB_API_URL=${F8A_JOB_API_URL:-http://bayesian-jobs-bayesian-preview.b6ff.rh-idev.openshiftapps.com};
+            F8A_API_V2_URL=${F8A_API_V2_URL:-https://f8a-analytics-preview-2445582058137.staging.gw.apicast.io};
+            F8A_GREMLIN_URL=${F8A_GREMLIN_URL:-http://bayesian-gremlin-http-preview-b6ff-bayesian-preview.b6ff.rh-idev.openshiftapps.com};
+            F8A_3SCALE_URL=${F8A_3SCALE_URL:-https://3scale-connect.api.prod-preview.openshift.io/};
+            F8A_THREE_SCALE_PREVIEW_URL=${F8A_THREE_SCALE_PREVIEW_URL:-https://f8a-analytics-preview-2445582058137.production.gw.apicast.io};
+            F8A_BACKBONE_API_URL=${F8A_BACKBONE_API_URL:-http://f8a-server-backbone-bayesian-preview.b6ff.rh-idev.openshiftapps.com/};
+            F8A_SERVICE_ID=${F8A_SERVICE_ID:-2555417755633};
+            F8A_GEMINI_API_URL=${F8A_GEMINI_API_URL:-https://gemini.api.prod-preview.openshift.io/};
+            F8A_LICENSE_SERVICE_URL=${F8A_LICENSE_SERVICE_URL:-https://license-analysis.api.prod-preview.openshift.io};
+            OSIO_AUTH_SERVICE=${OSIO_AUTH_SERVICE:-https://auth.prod-preview.openshift.io};
+            MANIFESTS_BUCKET_URL=${MANIFESTS_BUCKET_URL:-https://public-dynamic-manifests.s3.amazonaws.com/};
+        - name: Build Docker Image 
+          run: docker build -t f8a-e2e-tests .
```

Comment:
Do we really need a docker image to run e2e?

Reply:
yes this is how jenkins runs it but we can change only run the runtest.sh by exporting all vars also should work